news/2022-07-09-new-featured-artist-aran.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: aran"
+date: 2022-07-09 10:00:00 +0000
+---
+
+Are you ready for the challenge that is **aran**, our latest Featured Artist?
+
+![](https://assets.ppy.sh/artists/282/header.jpg)
+
+For many years, the foundational, genre-defying tunes of **aran** laid claim to one of the greatest challenges osu! has ever known in [rustbell](https://osu.ppy.sh/users/227717)'s much beloved set for [Graces of Heaven](https://osu.ppy.sh/beatmapsets/147910), which shook the beatmapping world to its core back in 2016. Beyond that, fans of *beatmania IIDX*, *jubeat* and *SOUND VOLTEX* are likely already familiar with the aural glory that encompasses his creations.
+
+Ever since, **aran**'s work has woven itself into the musical and cultural heart of osu! across a variety of remixes and original tracks, and it is now our absolute delight to announce today that **20** new tracks from him are now enshrined in our Featured Artist library forever more.
```

Comment:
```suggestion
Ever since, **aran**'s work has woven itself into the musical and cultural heart of osu! across a variety of remixes and original tracks, and it is now our absolute delight to announce today that **20** new tracks from him are now enshrined in our Featured Artist library forevermore.
```